README.md

```diff
@@ -824,15 +824,17 @@ variable in your project, when dbt searches for implementations of a dispatched
 `dbt_utils` macro, it will search through your listed packages _before_ using
 the implementations defined in `dbt_utils`.
 
-Set the variable:
+Set the property `var` in `dbt_project.yml`:
 ```yml
 vars:
   dbt_utils_dispatch_list:
-    - first_package_to_search    # likely the name of your root project
+    - my_analytics_proj          # likely the name of your root project (only the root folder)
```

Comment:
Could we keep this as `first_package_to_search`, for the sake of the example further down? I think it's fair to clarify that this is the `name:` property from `dbt_project.yml` of your root project.